karma.conf.js

```diff
@@ -55,6 +55,7 @@ module.exports = function(config) {
       'jujugui/static/gui/build/app/assets/javascripts/d3-min.js',
 
       'jujugui/static/gui/src/app/components/**/*.js',
+      'jujugui/static/gui/src/test/required-props.js',
```

Comment:
I think a comment here explaining that this needs to go ahead of any tests would be helpful in the future